layers/+tools/sailfish-scratchbox/packages.el

```diff
@@ -0,0 +1,39 @@
+;;; packages.el --- sailfish scratchbox layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2017 Victor Polevoy
+;;
+;; Author: Victor Polevoy <fx@thefx.co>
+;; URL: https://github.com/vityafx/sailfish-scratchbox-layer
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: MIT
+;;; Commentary:
+;;
+;; Commands
+;;
+;; SPC c s - sailfish scratchbox menu
+;; SPC c s b - sailfish scratchbox mb2 build
+;; SPC c s d - sailfish scratchbox deploy rpms
+;; SPC c s i - sailfish scratchbox install rpms
+
+;;; Code:
+
+(defconst sailfish-scratchbox-packages
+  '(sailfish-scratchbox))
+
+(defun sailfish-scratchbox/init-sailfish-scratchbox ()
+  (use-package sailfish-scratchbox
+    :defer t
+    :init
+    (progn
+      (spacemacs/declare-prefix "cs" "sailfish os scratchbox")
+      (spacemacs/set-leader-keys
+        "csb" 'sailfish-scratchbox-mb2-build
+        "csd" 'sailfish-scratchbox-deploy-rpms
+        "csi" 'sailfish-scratchbox-install-rpms
+      )
+    )
+  )
+)
+;;; packages.el ends here
```

Comment:
"ends here" comment is unnecessary - this file isn't ia package itself. Please remove.